gui/src/installer/step/mod.rs

```diff
@@ -63,39 +62,34 @@ pub struct Final {
     context: Option<Context>,
     warning: Option<String>,
     config_path: Option<PathBuf>,
-    hot_signer_fingerprint: Fingerprint,
-    hot_signer_is_not_used: bool,
 }
 
 impl Final {
-    pub fn new(hot_signer_fingerprint: Fingerprint) -> Self {
+    pub fn new() -> Self {
         Self {
             context: None,
             generating: false,
             warning: None,
             config_path: None,
-            hot_signer_fingerprint,
-            hot_signer_is_not_used: false,
         }
     }
 }
 
+impl Default for Final {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 impl Step for Final {
     fn load_context(&mut self, ctx: &Context) {
         self.context = Some(ctx.clone());
```

Comment:
Maybe no need to clone the whole context anymore, instead keep only the internal_bitcoind used line 106

Reply:
I've changed this now.